hive_agent/wallet/wallet.py

```diff
@@ -1,27 +1,41 @@
+import logging
+import sys
+from datetime import datetime
+
 from typing import Any, Dict, List
 
 from eth_account import Account
 from eth_account.datastructures import SignedMessage
 from eth_account.signers.local import LocalAccount
 from eth_account.messages import encode_defunct
 
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```

Comment:
keep this as INFO because we don't want to see the debug logs yet. we'll set the log level as environment variables.